### ISSUE TITLE
Duplicate errors for "space-before-function-parentheses"

### ISSUE THREAD

**Issue by feross** [eslint/eslint#1934]
With this rule:

```json
{
  "space-before-function-parentheses": [2, "always"]
}
```

And this code:

```js
function hey() {
  1 + 1
}
hey()
```

Two errors are reported:

```
  file.js:1:0: Function name "hey" must be followed by whitespace. (space-after-function-name)
  file.js:1:12: Missing space before function parentheses. (space-before-function-parentheses)
```

**Comment by lo1tuma** [comment]
It seems that you have also `space-after-function-name` enabled, which is now deprecated because it will be replaced by `space-before-function-parentheses`.

The warning from `space-before-function-parentheses` is correct, your example code doesn’t have a space before parentheses.

**Comment by feross** [comment]
Gah, I had an `.eslintrc` floating around in my home folder. My bad.
